lib/src/flexible_alert.cc

```diff
@@ -289,10 +294,9 @@ static void flexible_alert_evaluate(flexible_alert_t* self, rule_t* rule, const
         if (!ftymsg) {
             // some metrics are missing
             log_trace("abort evaluation of rule %s because %s metric is missing", rule_name(rule), topic);
-            std::stringstream ss;
-            ss << param << " = " << "NaN";
-            auditValues.push_back(ss.str());
+
             isMetricMissing = true;
+            auditValues += (auditValues.empty() ? "" : ", ") + auditValue(param, NULL);
```

Comment:
We lost the information NaN which is different of empty value

Reply:
Not a Not-a-Number here, but a missing value...